contrib/localnet/docker-compose.yml

```diff
@@ -120,6 +187,20 @@ services:
         ipv4_address: 172.20.0.100
     entrypoint: ["geth", "--dev", "--http", "--http.addr", "172.20.0.100", "--http.vhosts", "*", "--http.api", "eth,web3,net", "--http.corsdomain", "https://remix.ethereum.org", "--dev.period", "2"]
 
+  eth2:
+    build:
+      context: ./anvil
```

Comment:
AFAIK `anvil` doesn't support pre EIP 1559 transactions. So this is blocked by https://github.com/zeta-chain/node/pull/2387

Reply:
Also, should we remove `rosetta` image?

Reply:
> AFAIK anvil doesn't support pre EIP 1559 transactions. So this is blocked by https://github.com/zeta-chain/node/pull/2387

This is just refactoring, so if it was broken before this it will continue to be broken. Not a blocker if it was broken before imo.

> Also, should we remove rosetta image?

I don't specifically know of anything that uses it

Reply:
Yes it doesn't work but it's currently not used, the test for chain migration is disabled